BDArmory/Guidances/MissileGuidance.cs

```diff
@@ -393,6 +393,15 @@ public static Vector3 GetAirToGroundTarget(Vector3 targetPosition, Vector3 targe
             }
         }
 
+        public static Vector3 GetDLDeviation(Vector3 aamtarget,float drift=5)
```

Comment:
This will generate a flat distribution, which is likely not what you want. You probably want to just use
```csharp
aamTarget += drift * Time.fixedDeltaTime * VectorUtils.GaussianVector3();
```
where you're calling this function instead. I've added the helper `VectorUtils.GaussianVector3()` to the dev branch, which generates a `Vector3` with elements from an approximately normal distribution (mean 0, std.dev 1).